client/src/components/header/header.css

```diff
@@ -0,0 +1,32 @@
+body {
+    margin: 0;
+}
+
+.headerContainer {
```

Comment:
In your `src/App.css` you named `App-logo`, and now you're naming `headerContainer`? Which is it? Camel case or sneak case?